### PR TITLE
Update indexer-batch.md regarding last code changes

### PR DESCRIPTION
**Pull request by vzabaznov**
## Purpose of this pull request

This pull request (PR) updates batch sizes options regarding last code changes

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/extension-dev-guide/indexer-batch.html#eav-indexer-optimization

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.4/extension-dev-guide/indexer-batch.md [hunk 1]
```diff
@@ -144,6 +148,13 @@ return [
 ```
 
 The batches size for `catalog_category_product`, `catalogpermissions_category`, `catalogpermissions_category` will be set for all product types.
+
+Batch size for `catalogsearch_fulltext` can be set different parameters.
```
Comment (BarnyShergold) [inline src/guides/v2.4/extension-dev-guide/indexer-batch.md:152]:
```suggestion
Batch size for `catalogsearch_fulltext` can be set using different parameters.
```

File: src/guides/v2.4/extension-dev-guide/indexer-batch.md [hunk 2]
```diff
@@ -144,6 +148,13 @@ return [
...
+
+* `partial_reindex` - represent how many product will be proceeded in partial reindex.
```
Comment (BarnyShergold) [inline src/guides/v2.4/extension-dev-guide/indexer-batch.md:154]:
```suggestion
* `partial_reindex` - represents how many products will be processed in a partial reindex.
```

File: src/guides/v2.4/extension-dev-guide/indexer-batch.md [hunk 3]
```diff
@@ -144,6 +148,13 @@ return [
...
+* `elastic_save` - represent how many product will be saved as batch into Elastic.
```
Comment (BarnyShergold) [inline src/guides/v2.4/extension-dev-guide/indexer-batch.md:155]:
```suggestion
* `elastic_save` - represents how many products will be saved as a batch into Elasticsearch.
```

File: src/guides/v2.4/extension-dev-guide/indexer-batch.md [hunk 4]
```diff
@@ -144,6 +148,13 @@ return [
...
+* `mysql_get` - represent how many searchable product will be retrived from Mysql.
```
Comment (BarnyShergold) [inline src/guides/v2.4/extension-dev-guide/indexer-batch.md:156]: Is this needed as MySQL search was removed in 2.4?
```suggestion
* `mysql_get` - represents how many searchable products will be retrieved from Mysql.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Not sure why linting is failing. Please check after next commit.

**Review by dobooth — APPROVED** [review]

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Comment by dobooth** [comment]
running tests

**Pull request merged**